.travis.yml

```diff
@@ -67,7 +67,7 @@ before_install:
 install:
   - . $HOME/spack/share/spack/setup-env.sh
   - spack install gotcha@1.0.3 && spack load gotcha@1.0.3
-  - spack install margo@0.4.3^mercury@1.0.1+bmi~ofi~boostsys && spack load argobots && spack load mercury && spack load margo
+  - spack install mochi-margo ^libfabric fabrics=rxm,sockets,tcp && spack load argobots && spack load mercury && spack load mochi-margo
```

Comment:
For future reference and clarification when I update our spackage and build docs:

1. We now work with `^mercury+bmi~ofi`, `^mercury+ofi~bmi`, or both enabled, correct?
2. Do we conflict if `^mercury~bmi~ofi`?
3. Do we now require the specific  `^libfabric fabrics=rxm,sockets,tcp` dependency all the time? Only if `+ofi`? Or will we now always break with the default libfabric that Spack installs for Mercury?

Reply:
1. Yes (but I personally didn't test the both case - Hyogi did that awhile back)
2. Yes
3. That specific `libfabrics fabrics=rxm,sockets,tcp` config is necessary for my Docker builds and possibly Travis since they are both Ubuntu Bionic. However, Mercury v2.0 'ofi+tcp' requires `libfabrics fabrics=rxm,tcp`, but their spackage doesn't properly capture that dependency. The former spec will work on more machines, so it is probably what we want in our spackage.